src/components/handle.tsx

```diff
@@ -0,0 +1,63 @@
+import '../styles/handle.css';
+import React from 'react';
+
+const handleRotation = Symbol();
+const toggleMouseDown = Symbol();
```

Comment:
Why would you use `Symbol`s here? It really seems like a completely forced, overly-clever solution, to the wrong problem. Unless I'm missing something, there is no reason to use those over normal method names.

If you wanted privacy, note that `Symbol`s are not private. If you wanted to denote privacy, Typescript allows you to have `private` methods that make that explicit. Or, if you wanted, you could use the official `#method()` experimental proposal through Babel. But I would just stay with TS.

Reply:
https://medium.com/@martin_hotell/10-typescript-pro-tips-patterns-with-or-without-react-5799488d6680 ⬅️Bullet point 3

Just trying out new patterns 😬 But I agree, it feels a bit convoluted... We can change it back to private to be more explicit if you want. Come to think of it, I think I'd prefer to actually.

I mean, the private accessor is just to help us avoid mistakes while coding in this case . Everything on runtime happens client-side, so we don't really care if the end user messes around with the code.

Reply:
Fixed and pushed it to the PR branch 👍